index.html

```diff
@@ -75,69 +84,75 @@ <h4 class="about-head">Tonic</h4>
       </div>
     </div>
+    </div>
     <div class="about-section">
-      <img src="./img/Snapshoot Portfolio (1).png" class="card-img" />
-      <h4 class="about-head">Multi-Post <br> Stories</h4>
+      <img src="./img/Dbg2.png" class="card-img" />
       <div class="canopy">
+        <h4 class="about-head">Multi-Post Stories</h4>
         <ul class="cano">
-          <li class="tec">CANOPY</li>
-          <li class="tech">Back End Dev</li>
+          <li class="tec">FACEBOOK</li>
+          <li class="tech">Full Stack Dev</li>
           <li class="tech">2015</li>
         </ul>
         <p class="canopy-txt">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required
+          Experimental content creation feature that allows users to add to an existing story over 
+          the course of a day without spamming their friends.
         </p>
         <ul class="tech1">
           <li class="tech-t">HTML</li>
+          <li class="tech-r">Ruby On Rails</li>
           <li class="tech-t">CSS</li>
           <li class="tech-j">JavaScript</li>
         </ul>
         <button type="button" class="project-btn">See Project</button>
       </div>
     </div>
     <div class="about-section">
-      <img src="./img/Snapshoot Portfolio (2).png" class="card-img" />
-      <h4 class="about-head">Tonic</h4>
+      <img src="./img/Dbg3.png" class="card-img" />
       <div class="canopy">
+        <h4 class="about-head">Facebook 360</h4>
         <ul class="cano">
-          <li class="tec">CANOPY</li>
-          <li class="tech">Back End Dev</li>
+          <li class="tec">FACEBOOK</li>
+          <li class="tech">Full Stack Dev</li>
           <li class="tech">2015</li>
         </ul>
         <p class="canopy-txt">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required
+          Exploring the future of media in Facebook's
+           first Virtual Reality app; a place to discover and 
+          enjoy 360 photos and videos on Gear VR.
         </p>
         <ul class="tech1">
           <li class="tech-t">HTML</li>
+          <li class="tech-r">Ruby On Rails</li>
           <li class="tech-t">CSS</li>
           <li class="tech-j">JavaScript</li>
         </ul>
         <button type="button" class="project-btn">See Project</button>
       </div>
     </div>
     <div class="about-section">
-      <img src="./img/Snapshoot Portfolio (3).png" class="card-img" />
-      <h4 class="about-head">Multi-Post <br> Stories</h4>
+      <img src="./img/Dbg4.png" class="card-img" />
       <div class="canopy">
+        <h4 class="about-head">Uber Navigation</h4>
         <ul class="cano">
-          <li class="tec">CANOPY</li>
-          <li class="tech">Back End Dev</li>
-          <li class="tech">2015</li>
+          <li class="tec">UBER</li>
+          <li class="tech"> Lead Developer</li>
+          <li class="tech">2018</li>
         </ul>
```

Comment:
- [ ] Please check and remove the extra space in your project details section. 👍🏿 

### Your design ⬇️ 
![image](https://github.com/Danny-imo/my-project/assets/117144028/17ce4be4-2373-4eef-a740-5bd8991327b6)


### Figma design ⬇️ 
![image](https://github.com/Danny-imo/my-project/assets/117144028/1ea3d592-850a-47a2-a9a3-bdda308aa6f3)

Reply:
[OPTIONAL] Please ensure that you always match your design according to the given design kindly check your `Uber` font weight, Consistency is crucial in design, especially when working on a larger project or within an established brand. Matching your design to the given design ensures a cohesive visual identity across different components, screens, or materials. Consistency helps create a unified experience for users and maintains brand recognition and professionalism. :slightly_smiling_face: 

Below is an example of the difference: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![s uber navigation](https://github.com/Danny-imo/my-project/assets/43528594/a987c33f-138e-4174-8a11-0193627dfc9e)

</td>
      <td>

![Figma Design navigation](https://github.com/Danny-imo/my-project/assets/43528594/abca9453-ca98-408d-b02f-d273994fba51)

</td>
    </tr>
  </table>